.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: java
+
+dist: trusty
+
+jdk:
+  - oraclejdk8
+
+install: true
+
+script: make build
+
+deploy:
+  skip_cleanup: true
+  script: make deploy
+
+notifications:
+  email:
+    recipients:
```

Comment:
How to become the member of this group notification `stream-registry-committers@groups.homeawaycorp.com ?